src/js/modules/accounts/components/forms/SubscriptionForm.js

```diff
@@ -60,42 +51,21 @@ const validate = formValues => {
   return errors;
 };
 
-const renderField = ({
-  input,
-  label,
-  type,
+const SubscriptionForm = ({
   classes,
-  meta: { touched, error, warning },
+  handleSubmit,
+  submitting,
+  callToAction,
```

Comment:
What is this? Also, is this a reduxForm name or your name?

Reply:
meta, submitting, etc. are all reduxForm.

Reply:
No, callToAction